gubernator/static/style.css

```diff
@@ -1,25 +1,30 @@
 body {
-    font-family: sans-serif;
+    font-family: 'Roboto', sans-serif !important;
     padding: 0;
     margin: 0;
     scroll-behavior: smooth;
 }
 #header {
+    align-items: center;
```

Comment:
Please don't change this. this breaks the build page styling and is irrelevant for this PR.

<img width="705" alt="screen shot 2018-02-16 at 1 23 09 pm" src="https://user-images.githubusercontent.com/211868/36329922-95299524-131c-11e8-9217-3ae18891bbf8.png">